knowhere/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@
 include_directories(${INDEX_SOURCE_DIR}/knowhere)
 include_directories(${INDEX_SOURCE_DIR}/thirdparty)
 include_directories(${INDEX_SOURCE_DIR}/thirdparty/NGT/lib)
+include_directories(${INDEX_SOURCE_DIR}/knowhere/knowhere/utils)
```

Comment:
we already have a utils dir in ${INDEX_SOURCE_DIR}/knowhere， so suggest moving these files into this dir

Reply:
should have one utils dir for knowhere

Reply:
the file Error.h/Json.h/Status.cpp/Status.h under knowhere/utils should be moved to knowhere/knowhere/common, I'll do that using another PR